scene/resources/font.cpp

```diff
@@ -1526,798 +1546,947 @@ void FontData::remove_cache(int p_cache_index) {
+	ClassDB::bind_method(D_METHOD("draw_multiline_string_outline", "canvas_item", "pos", "text", "alignment", "width", "font_size", "max_lines", "size", "modulate", "flags", "direction", "orientation"), &FontConfig::draw_multiline_string_outline, DEFVAL(HORIZONTAL_ALIGNMENT_LEFT), DEFVAL(-1), DEFVAL(DEFAULT_FONT_SIZE), DEFVAL(-1), DEFVAL(1), DEFVAL(Color(1.0, 1.0, 1.0)), DEFVAL(TextServer::BREAK_MANDATORY | TextServer::BREAK_WORD_BOUND | TextServer::JUSTIFICATION_KASHIDA | TextServer::JUSTIFICATION_WORD_BOUND), DEFVAL(TextServer::DIRECTION_AUTO), DEFVAL(TextServer::ORIENTATION_HORIZONTAL));
+
+	ClassDB::bind_method(D_METHOD("has_char", "char"), &FontConfig::has_char);
+	ClassDB::bind_method(D_METHOD("get_supported_chars"), &FontConfig::get_supported_chars);
+
+	ClassDB::bind_method(D_METHOD("get_char_size", "char"), &FontConfig::get_char_size);
+	ClassDB::bind_method(D_METHOD("draw_char", "canvas_item", "pos", "char", "modulate"), &FontConfig::draw_char, DEFVAL(Color(1.0, 1.0, 1.0)));
+	ClassDB::bind_method(D_METHOD("draw_char_outline", "canvas_item", "pos", "char", "size", "modulate"), &FontConfig::draw_char_outline, DEFVAL(-1), DEFVAL(Color(1.0, 1.0, 1.0)));
+
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "font", PROPERTY_HINT_RESOURCE_TYPE, "Font"), "set_font", "get_font");
 
-	ClassDB::bind_method(D_METHOD("update_changes"), &Font::update_changes);
+	ADD_GROUP("Variation", "variation");
+	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "variation_opentype"), "set_variation_opentype", "get_variation_opentype");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "variation_embolden", PROPERTY_HINT_RANGE, "-2,2,0.01"), "set_variation_embolden", "get_variation_embolden");
+	ADD_PROPERTY(PropertyInfo(Variant::TRANSFORM2D, "variation_transform"), "set_variation_transform", "get_variation_transform");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "variation_customize_fallbacks"), "set_variation_customize_fallbacks", "get_variation_customize_fallbacks");
 
-	ClassDB::bind_method(D_METHOD("get_rids"), &Font::get_rids);
+	ADD_GROUP("OpenType Features", "opentype");
+	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "opentype_features"), "set_opentype_features", "get_opentype_features");
+
+	ADD_GROUP("Extra Spacing", "spacing");
+	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_glyph"), "set_spacing", "get_spacing", TextServer::SPACING_GLYPH);
+	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_space"), "set_spacing", "get_spacing", TextServer::SPACING_SPACE);
+	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_top"), "set_spacing", "get_spacing", TextServer::SPACING_TOP);
+	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_bottom"), "set_spacing", "get_spacing", TextServer::SPACING_BOTTOM);
```

Comment:
```suggestion
	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_glyph", PROPERTY_HINT_NONE, "suffix:px"), "set_spacing", "get_spacing", TextServer::SPACING_GLYPH);
	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_space", PROPERTY_HINT_NONE, "suffix:px"), "set_spacing", "get_spacing", TextServer::SPACING_SPACE);
	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_top", PROPERTY_HINT_NONE, "suffix:px"), "set_spacing", "get_spacing", TextServer::SPACING_TOP);
	ADD_PROPERTYI(PropertyInfo(Variant::INT, "spacing_bottom", PROPERTY_HINT_NONE, "suffix:px"), "set_spacing", "get_spacing", TextServer::SPACING_BOTTOM);
```

Adds suffixes/units to properties (see https://github.com/godotengine/godot/pull/50106, https://github.com/godotengine/godot/pull/61240, and https://github.com/godotengine/godot/pull/50009)